index.js

```diff
@@ -33,7 +33,7 @@ async function parseRedirects(event) {
 function checkRedirect(request) {
   const url = new URL(request.url).pathname
   for (const [pattern, redirectUrl] of redirectMap) {
-    if (url.match(pattern)) {
+    if (url.match(pattern) && pattern != "") {
```

Comment:
Itd be better to check if the string is empty or not before performing the regex.

Reply:
I think this should suffice, if not please feel free to raise a PR!

Reply:
Raise a PR to fix this PR?

Reply:
done!